### PR TITLE
Errata 2.5.3

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/26153

### CODE REVIEW

**Comment by oafischer** [comment]
One more errata came through, adding to PR now.

File: release_notes/errata.adoc [hunk 1]
```diff
@@ -11,6 +11,8 @@ FIPS notice: If you do not specify your own ciphers in `spec.ingress.sslCiphers`
 
 * Fixes a permission issue when using the unsupported `--validate-cluster-security` flag as a `HypershiftDeployment` controller argument. (https://bugzilla.redhat.com/show_bug.cgi?id=2109544[Bugzilla 2109544])
 
+* Updates the search aggregator logic to avoid concurrent sync requests from a managed cluster. (https://bugzilla.redhat.com/show_bug.cgi?id=2092863[Bugzilla 2092863])
+
```
Comment (cadawson) [inline release_notes/errata.adoc:15]: Can we add this statement as a new bullet: 
* Delivers updates to one or more of the product container images and security fixes. ?

By the time we release the z-streams there are usually one or more container updates in the package. This just tells the customers that they are there. Thanks!

Comment (oafischer) [reply release_notes/errata.adoc:15]: Added it, thanks!

**Review by dockerymick — APPROVED** [review]
I see you added Chris' suggestion. Looks good , thanks Oliver !

**Pull request merged**